### PR TITLE
Feature1: Animal table

### PR DESCRIPTION
**Pull request by ecekpo**
--Added changes in the query --
-Attached screenshot
Update query 1
Update query 3
Update query2
Update query4

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,36 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+//Find all animals whose name ends in "mon".
+SELECT * FROM animals
+WHERE name LIKE '%mon';
+
+
+//List the name of all animals born between 2016 and 2019.
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```
Comment (Deepakdanger) [inline queries.sql]: - [ ] Here we have to select name.
```
SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12';
```

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,36 @@
...
+
+
+//List the name of all animals that are neutered and have less than 3 escape attempts.
+SELECT * FROM animals WHERE neutered='True' AND escape_attempts<3;
```
Comment (Deepakdanger) [inline queries.sql]: - [ ] Here we have to select name.

`SELECT name FROM animals WHERE neutered = 'true' AND escape_attempts < '3';`

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,36 @@
...
+
+
+//List the date of birth of all animals named either "Agumon" or "Pikachu".
+SELECT * FROM animals WHERE name='Agumon' OR name='Pikachu';
```
Comment (Deepakdanger) [inline queries.sql]: - [ ] Here we have to select the `date_of_birth`.
`SELECT date_of_birth FROM animals WHERE name = 'Pikachu' OR name = 'Agumon';`

File: queries.sql [hunk 4]
```diff
@@ -1,3 +1,36 @@
...
+
+
+//List name and escape attempts of animals that weigh more than 10.5kg
+SELECT * FROM animals WHERE weight_kg>10.5;
```
Comment (Deepakdanger) [inline queries.sql]: - [ ] Here we have to select the name & escape_attempts.
`SELECT name, escape_attempts FROM animals WHERE weight_kg > '10.5';`

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @ecekpo , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (ecekpo) [reply queries.sql]: Hello Deepakdanger,
Thank you for the review but when I used the above command, this is the error message gotten. Find attached.
Error1 message

File: queries.sql (hunk 2 above)
Comment (ecekpo) [reply queries.sql]: Hello Deepakdanger,
This very one nothing display after I ran the command given above. What should be done please?
No output

File: queries.sql (hunk 3 above)
Comment (ecekpo) [reply queries.sql]: Hi Deepakdanger,
The same thing, no display of anything after running the above command given. Find attached
No output2

File: queries.sql (hunk 4 above)
Comment (ecekpo) [reply queries.sql]: Hello Deepakdanger,
The same thing with the above command ran, with 
No output3
no display of anything. Find attached. what should I do, pls.

**Review by AnselemOdims — APPROVED** [review]
### Hi  @ecekpo 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**